### DISCUSSION TITLE
Use Stylelint to improve our CSS

### DISCUSSION THREAD

**Discussion by aidamag** [dxc-technology/halstack-react#1369] [Ideas]
I have been checking out Stylelint and it looks like a very good tool to improve our CSS. It is used to report any bad code or errors and it will help us to write maintainable code following the conventions we want (it can be highly customized with rules).

Styled components has this feature so it would not be a problem to install it. 
https://styled-components.com/docs/tooling#stylelint

This is its website: https://stylelint.io/

**Comment by GomezIvann** [comment]
This can be very interesting Aida, maybe we can add this as an epic to the backlog ("HTML and CSS validation tools"). I heard about this in the past about this library seems to be well-accepted by the community.

**Comment by raquelarrojo** [comment]
I didn't know about it, but it seems to be useful. We can give it a try.

**Comment by marcialfps** [comment]
I think all these tools that help to have maintainable code are well-received. I agree with the idea ✌🏼
